README.md

```diff
@@ -74,6 +97,10 @@ Anyone and everyone is welcome to [contribute](https://github.com/Esri/mapbook-w
 3. Contribute code
 4. Improve documentation
 
+## MDTOC
+
+Generation of this and other document's table of contents in this repository was performed using the [MDTOC package for Atom](https://atom.io/packages/atom-mdtoc).
```

Comment:
I stuck the apostrophe in the wrong location in the initial draft of this message. Easy fix -

```suggestion
Generation of this and other documents' table of contents in this repository was performed using the [MDTOC package for Atom](https://atom.io/packages/atom-mdtoc).
```